DataLayer/CmuData/CmuData.cpp

```diff
@@ -11,52 +11,52 @@ CmuData::~CmuData()
 
 Cmu CmuData::cmuZero() const
 {
-    return cmuList[0];
+    return cmuList_.at(0);
 }
 Cmu CmuData::cmuOne() const
 {
-    return cmuList[1];
+    return cmuList_.at(1);
 }
 Cmu CmuData::cmuTwo() const
 {
-    return cmuList[2];
+    return cmuList_.at(2);
 }
 Cmu CmuData::cmuThree() const
 {
-    return cmuList[3];
+    return cmuList_.at(3);
 }
 
 void CmuData::setCmuZero(Cmu cmuZero)
 {
-    cmuList[0] = cmuZero;
```

Comment:
We can't really use `append`, since it'll attach `cmuzero` to the very end of the list so `cmuList_.at(0)` will still get the same value. Maybe use `replace` coupled with `reserve` instead? This goes for the rest of the lists.